packages/iot-app-kit-visualizations/src/index.ts

```diff
@@ -1,2 +1,4 @@
+export * from './constants';
```

Comment:
Can we do named exports for this one? Don't want to export something extra on accident.